t/webservice.t

```diff
@@ -158,19 +158,15 @@ transactionally {
     };
 
     subtest "store-data" => {
-        my %data = %(
-            :dataset_name('MyNewTestDataset'),
-            :data_var('PlantProduction::AgriculturalArea::agricultural_area'),
-            :data_val(42)
-        );
-        ok $ws.store-data($user, %data), "Simple input stored";
-
-        %data = %(
-            :dataset_name('MyNewTestDataset'),
-            :data_var('Livestock::DairyCow[MK]::Excretion::dairy_cows'),
-            :data_val(42)
-        );
-        ok $ws.store-data($user, %data), "Instance input stored";
+        my $dataset = 'MyNewTestDataset';
+        my $var = 'PlantProduction::AgriculturalArea::agricultural_area';
+        my $value = 42;
+        ok $ws.store-data($user,  :$dataset, :$var, :$value), "Simple input stored";
```

Comment:
This form would not be callable today, for the reason mentioned earlier: it'd try to pass an undefined value for the branches rather than leaving it alone.